components/tekton-ci/repository.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: gitops
+spec:
+  url: "https://github.com/redhat-appstudio/managed-gitops"
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: service-provider-integration
+spec:
+  url: "https://github.com/redhat-appstudio/service-provider-integration-operator"
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
+metadata:
+  name: e2e-tests
+spec:
+  url: "https://github.com/redhat-appstudio/e2e-tests"
```

Comment:
This is included by mistake? What would e2e folder contain after this change? Is it needed? (maybe in e2e repository?)
If not, need to remove this block as well as the e2e folder under components.

Reply:
folder in components removed, but this code is needed, to be able to run pipelines defined in https://github.com/redhat-appstudio/e2e-tests/tree/main/.tekton